README.md

```diff
@@ -144,12 +144,12 @@ class AdComponent extends React.Component {
   render() {
     return (
       <View>
-        <AdChoicesView style={{ position: 'absolute', left: 0, top: 0 }} />
         <AdIconView style={{ width: 50, height: 50 }} />
         <MediaView style={{ width: 160, height: 90 }} />
         <TriggerableView>
           <Text>{this.props.nativeAd.description}</Text>
         </TriggerableView>
+        <AdChoicesView style={{backgroundColor:'#fff'}}  location={"bottomRight"} expandable={true} /> // Should be at bottom !important  
```

Comment:
Please remove the comment - AdChoicesView doesn't really have to be at the bottom, it depends on how the user styles it and the overall layout it's in.

Reply:
this is because of `zIndex` issue on IOS.

Reply:
offset` issue occurs on ios when using `position:"absolute"`

Reply:
I can't recreate the zIndex issue you're referring to. Could you provide a reproduction please?

Reply:
`zIndex` is occurring when adView is in list format `topLeft` & `bottomLeft` side where AdIcon is overlapping, the adChoice in position `absolute` and adIcon is written at top of the child content, that's how you can reproduce the error. 

So according to my testing `adIcon` & `mediaView` will overlap the adchoice when (IOS only)

- Adicon is written at the top
- AdIcon styleProp is `position:absolute`

I also created offset issue. On some scenarios.

Reply:
That doesn't mean the AdChoicesView has to be last, it's simply how absolute positioning works and it's not specific to AdChoices. When items are absolutely positioned the last item gets to be frontmost. This isn't an issue with this library rather how layout works, so we can remove that comment.

Reply:
Ok as it is using position absolute by default. 
I'm going to add a note to prevent developers scratching heads.

Reply:
The current AdChoices style does not include an absolute position. However I did encounter some issues with the height & width of the view no measuring correctly and had to specify those manually - I think it's related to the issue you're having here as incorrectly measured views can easily overlap with each other.

Reply:
```js
<View style={[{flexDirection: 'row'}, this.props.style]} >
    <AdChoicesView location={ "topLeft"} expandable={false} /> // Ios&android 'zIndex' issue
    <AdIconView style={{
        width: 80,
        height: 80,
        borderRadius: 0,
        resizeMode: 'contain'
    }}/>
    <View style={{
        justifyContent: 'flex-start',
        paddingHorizontal: 10,
        paddingVertical: 0,
        flex: 1,
    }}>
        <Text style={styles.sponsorTitle}>{this.props.nativeAd.sponsoredTranslation}</Text>
        <TriggerableView>
            <Text style={styles.adTitle}>{trimText(this.props.nativeAd.advertiserName, 20)}</Text>
        </TriggerableView>
        <Text style={styles.socialContext}>{this.props.nativeAd.socialContext}</Text>
        <View style={styles.actionContainer}>
            <TriggerableView style={styles.button}>
                <Text style={styles.buttonText}>{this.props.nativeAd.callToActionText}</Text>
            </TriggerableView>
        </View>
    </View>
</View>
```

![screenshot 2018-11-02 at 8 20 49 pm](https://user-images.githubusercontent.com/22210490/47922467-2aae3a00-dedd-11e8-81c2-1cfa036e6854.png)

Reply:
@Suraj-Tiwari Try setting AdChoicesView `height: 22`. The image you've shown exactly illustrates what I said earlier that the view isn't measured correctly, so other views simply layout over it partially within it's container.

Set `style: {{height:22}}` and test again please.

Reply:
@erandagan Seriously man????? Seriously??? 
Adchoice isn't supposed to define with height, width or anything other than positioning value anything other than position is managed by itself. 

I wonder what exactly is issue for you about this commit?

Adicon and MediaView have 2 layers of media (Default from Fbaudiencesdk)
With Media View one is Normal and second one is blurry to fill rest of the blank space in case of media size is larger or smaller than defined height width.

Reply:
What I'm saying is that you're chasing the wrong problem here. The root cause for your "zIndex" issue is actually AdChoices being incorrectly measured.

Please keep a polite discussion - this is very rude of you to respond in that way while I'm trying to help.

Reply:
I'm extremely sorry for my rude behaviour, the time when i recreated this library i thought to make it simple and fast to integrate, keeping in mind that newcomers gonna use it too. 

if `position:absolute` cause view issue for some developers adding `// If facing zIndex issue place it to bottom` as help comment is wrong? because adding `style={{height: 22}} or style={{height:22, width: 80}}` doesn't resolve this issue at all,

as you said `height: 22` i'm sure you are referring to  `android` and i'm talking about `ios` and second thing `height: 22` is already [here](https://github.com/callstack/react-native-fbads/blob/e5933f68b9115d208af648d8ad66a417f122b761/src/native-ads/AdChoicesManager.tsx#L59) so no need to define it again until overriding `stylesheet height` property

Something is wrong on your side while checking. Can you confirm you've pull this pr to local while testing it out?

Reply:
When position is `absolute` then yes - you can add a comment stating that. But remember that this isn't really related to this library, it's just how absolute positioning works in general.

The issue I was referring to is slightly _different_. When position *!=* absolute, there is a bug that AdChoices isn't automatically measured correctly and you have to fallback into setting it's height manually. I guess we'll have to investigate further on that.

Reply:
Well found the reason of confusion, I've added `position absolute` in this commit which was previously removed , 
it is added in support of `topLeft`,`topRight`,`bottomLeft`,`bottomRight` etc. 

If anyone want to override he/she can simply pass style so, if this is okay. Then pr have no more issue

Reply:
> The issue I was referring to is slightly different. When position != absolute, there is a bug that AdChoices isn't automatically measured correctly and you have to fallback into setting it's height manually. I guess we'll have to investigate further on that.

I will have a look at it tomorrow what is causing this issue